dashboard/src/components/AppView/AppView.tsx

```diff
@@ -113,29 +115,38 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
 
     const sockets: WebSocket[] = [];
     let secrets = {};
+    let deployments = {};
```

Comment:
what are these for? aren't we already setting them in the state in `handleEvent`?

Reply:
I am assuming that it is to set an initial state based on the value in the manifest?

If that's the case, I'd add a comment.